app/controllers/api/v1/parks_controller.rb

```diff
@@ -1,39 +1,37 @@
-
 class Api::V1::ParksController < ApplicationController
-    protect_from_forgery unless: -> { request.format.json? }
-    def index
-        render json: {parks: Park.all }
-    end
-
-    def show
-        render json: { park: Park.find(params[:id]) }
-    end
-
-    def create
-        park = Park.new(name: params["name"], address: params["address"], city: params["city"], state: params["state"], zip: params["zip"], description: params["description"])
-        if park.save
-            render json: { park: park, error: [], successStatus: "Submitted Successfully!"}
-        else
-            render json: { park: {}, error: park.errors.full_messages, successStatus: "" }
-        end
-    end
-
-    def edit
+  protect_from_forgery unless: -> { request.format.json? }
+  def index
+      render json: {parks: Park.all }
+  end
+
+  def show
+      render json: { park: Park.find(params[:id]) }
+  end
+
+  def create
+    park = Park.new(name: params["name"], address: params["address"], city: params["city"], state: params["state"], zip: params["zip"], description: params["description"])
+    if park.save
+        render json: { park: park, error: [], successStatus: "Submitted Successfully!"}
+    else
+        render json: { park: {}, error: park.errors.full_messages, successStatus: "" }
     end
+  end
 
-    def update
+  def edit
+  end
 
-      park = Park.find(params[:id])
-      if park.update(park_params)
-        render json: {park: park, error: ''}
-      else
-        render json: {errors: park.errors}, status: 422
-      end
+  def update
+    park = Park.find(params[:id])
+    if park.update(park_params)
+      render json: {park: park, error: ''}
+    else
+      render json: {errors: park.errors}, status: 422
```

Comment:
It looks like you have differences between a key of `error` or `errors` in your code - I would keep this consistent with `errors`!